pact-jvm-consumer-groovy/src/test/groovy/au/com/dius/pact/consumer/groovy/PactResultSpec.groovy

```diff
@@ -134,7 +134,7 @@ class PactResultSpec extends Specification {
           '\tpath: /path\n' +
           '\tquery: [:]\n' +
           '\theaders: [Content-Type:application/json]\n' +
-          '\tmatchers: MatchingRules(rules=[body:Category(name=body, matchingRules={}), path:Category(name=path, matchingRules={})])\n' +
+          '\tmatchers: MatchingRules(rules={body=Category(name=body, matchingRules={}), path=Category(name=path, matchingRules={})})\n' +
```

Comment:
`.toString` formatting is different in kotlin.

Reply:
This does raise the issue of the output changing, which is mostly used in logging. Hopefully nobody depends on the format of the matchers which are output to the log files.